Profile-Card-2/index.html

```diff
@@ -1,38 +1,42 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
```

Comment:
Why are you making changes in someone else file, without creating an issue? If you want to improve someone else code first create an issue about it [here](https://github.com/mudit023/templates/issues) and then work on it if it gets assigned to you.

Reply:
I am sorry

Reply:
But I created one pull request with multiple commits

You can accept one